polyply/src/meta_molecule.py

```diff
@@ -226,6 +228,13 @@ def from_json(cls, force_field, json_file, mol_name):
 
         for node in nodes:
             attrs = init_graph.nodes[node]
+            if "resid" not in attrs:
+                LOGGER.warning("Node {} has no resid. Setting resid to {} + 1.".format(node, node))
```

Comment:
```suggestion
                LOGGER.warning("Node {} has no resid. Setting resid to {} + 1.", node, node)
```
And maybe 'Node 30271' is not very interpretable, but I'm splitting hairs here.